assets/js/pages/SapSystemsOverviewPage/SapSystemsOverview.test.jsx

```diff
@@ -80,13 +84,93 @@ describe('SapSystemsOverviews component', () => {
         tenant
       );
       expect(mainRow.querySelector('td:nth-child(5)')).toHaveTextContent(
-        dbAddress
+        sapSystemType
       );
       expect(mainRow.querySelector('td:nth-child(6)')).toHaveTextContent(
+        dbAddress
+      );
+      expect(mainRow.querySelector('td:nth-child(7)')).toHaveTextContent(
         'ENSA1'
       );
     });
 
+    it('should display the correct SAP system type', () => {
+      const sapSystemTypes = [
+        'ABAP',
+        'J2EE',
+        'ABAP|J2EE',
+        'J2EE|ABAP',
+        'SOME_SAP_SYSTEM_FEATURE|NOT_A_REAL_SYSTEM',
+      ];
+
+      const expectedSapSystemTypes = [
+        'ABAP',
+        'JAVA',
+        'ABAP/JAVA',
+        'ABAP/JAVA',
+        '',
+      ];
+      const sapSystems = [
```

Comment:
Maybe
```
const sapSystems = sapSystemTypes.map((type) => sapSystemFactory.build({ sap_system_type: type }))
```

Reply:
Factory is returned to original state, this is obsolete now, but yeah map would be here the way!